lib/tasks/copy_taxon_title.rake

```diff
@@ -0,0 +1,12 @@
+desc "copy taxons title field to internal name"
+task copy_taxons_title: :environment do
```

Comment:
This looks good. I was under the impression the Publishing API did this already. 

It looks like they do it for [linkables](https://github.com/alphagov/publishing-api/blob/012cb3f1ceb3b18e7059a367cc4030aa0763afb4/app/presenters/queries/linkable_presenter.rb#L10), but I can't see it being done for `get_content_items`: 

https://github.com/alphagov/publishing-api/blob/012cb3f1ceb3b18e7059a367cc4030aa0763afb4/app/presenters/queries/content_item_presenter.rb

It might be good to ask them why that is before we run this rake task.